src/Feedback/index.js

```diff
@@ -83,6 +83,7 @@ const Feedback = ({
   email,
   enableFeedbackText = true,
   enableEmoji = true,
+  openTop = false,
```

Comment:
You need to be aware that this should handle cases aside from the dashboard, so positioning elements based on a more generic prop is probably more appropriate.